### ISSUE TITLE
Squares view.

### ISSUE THREAD

**Issue by extempl** [Some1Else/life#2]
@Some1Else For squares you could use 1px gif with `width: 100%`: https://jsfiddle.net/extempl/t5hamwo3/
Also, try generate all html, it will look much cleaner.

**Comment by extempl** [comment]
Possibly, this would also look better with table use - then you will need to put those images only on the very top and left cells.

**Comment by some1else** [comment]
Yeah the base data structure could fit in a table. Thanks for the feedback
